status-monitor.lic

```diff
@@ -139,10 +139,21 @@ class GameFilter
           match = match.gsub(/\s/,'')
           if @valid_commands.include?(match)
             pause 3
-            fput(match)
+            fput(match.downcase)
             pause 5
           end
         end
+
+    ## Look for mixed-case prompts like J_u_M_p
+    tokens = line.split(" ")
```

Comment:
Some method chaining and/or Rubocop might make this a bit clearer, but that's a style thing